GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -24,6 +24,7 @@ public GVFSPlatform(string executableExtension, string installerExtension)
         public virtual bool IsUnderConstruction { get; } = false;
         public virtual bool SupportsGVFSService { get; } = true;
         public virtual bool SupportsGVFSUpgrade { get; } = true;
+        public virtual bool GitCoreFileMode { get; } = true;

```

Comment:
I would drop the default value here. This is something that we should decide explicitly on each platform.

Reply:
Also, I would move this to `IFileSystem`, as it's more a property of how the files are stored than just a general property of the platform.

Reply:
Should we just use the property that is already there `SupportsFileMode`?  Seems like that would mean the same thing?
CC: @wilbaker

Reply:
Yep that's what it was for :-). So definitely drop this one.

Reply:
Yeah that sounds right to me too